DB_Models_and_Eloquent.md

```diff
@@ -304,6 +304,22 @@ $users = User::all()->groupBy(function($item) {
 ### Never Update the Column
 
 If you have DB column which you want to be set only once and never updated again, you can set that restriction on Eloquent Model, with a mutator:
+
+- In version 9 and above:
+```php
+use Illuminate\Database\Eloquent\Casts\Attribute;
+
+class User extends Model
+{
+    protected function email(): Attribute
+    {
+        return Attribute::make(
+            set: fn ($value) => $this->email ?? $value,
```

Comment:
```suggestion
            set: fn ($value, $attributes) => $attributes['email'] ?? $value,
```
Hitting the accessor within a mutator is generally a bad idea. 
It doesn't matter much in this example but it may mislead some people if not corrected.